metricbeat/module/kubernetes/state_namespace/_meta/fields.yml

```diff
@@ -0,0 +1,23 @@
+- name: state_namespace
```

Comment:
I would prefer a name like `namespace_state` which reflects that this section gives information related to the namespace's state. `state_*` is quite an internal/elastic terminology and might not be so accurate/specific.

Note that I'm only referring to the fields' prefix and not the name of the metricset. The name of the metricset is just fine as is. But maybe we can improve the field's naming with what I mention.

@mlunadia do you have any preference here? `state_namespace.*` VS `namespace_state.*`?

Reply:
@gizas since you were the one suggesting this name, what do you think?

Reply:
I am also +1 for using `namespace_state` as the object field name.